src/redux/drawer/drawerSlice.ts

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from "@reduxjs/toolkit"
+import type { RootState } from "../store"
+
+export const initialState = {
+  drawer: false,
+}
+export const drawerSlice = createSlice({
+  name: "drawer",
+  initialState,
+  reducers: {
+    handleDrawer: (state) => {
+      state.drawer = !state.drawer
+    },
+  },
+})
+
+// Action creators are generated for each case reducer function
+export const { handleDrawer } = drawerSlice.actions
+
+export const selectOauth = (state: RootState) => state.allDrawer
```

Comment:
This is me being new to RTK too :) , but is this needed? Or should it be something like `selectDrawer`? 🤔

Reply:
Oops! Yes!